src/scripts.js

```diff
@@ -200,6 +211,19 @@ const autoCloseMenu = () => {
     searchIcon.classList.remove('disabled')
   }
 }
+const loadPantryPage = () => {
+  loadPage(pantryPage)
+  document.querySelector(".pantry-list").innerHTML = ""
+  currentUser.pantry.forEach(item => {
+    document.querySelector(".pantry-list").innerHTML += `
+    <tr class="pantry-table-row">
+      <td class="pantry-list-item">${capitalizeWords(item.name)}</td>
+      <td class="pantry-list-quantity">${item.quantity}</td>
+    <tr>
+    `
+  })
+}
+
 
```

Comment:
Should be incorporate units into this display? Should just be a simple case of adding another <td> element